microservices/nris_api/backend/README.md

```diff
@@ -0,0 +1,81 @@
+# Backend API
+
+The project uses a Python 3.6 runtime environment and [Flask REST-plus](https://flask-restplus.readthedocs.io/en/stable/) framework for the API.
+
+The application uses SQLAlchemy as our ORM to interact with the database.
+
+For the application directory structure, see [flask-RESTplus quickstart guide.](https://flask-restplus.readthedocs.io/en/stable/quickstart.html)
+
+## Directory/Naming convention
+
+The application directory is structured as follows:
+
+```
+|-- app
+    |-- api
+        |-- NAMESPACE_MODULE_NAME
+            |-- Namespace (Contains api namespace)
+            |-- MODULE_NAME
+                |-- Models (Contains all the database model definition used by SQLAlchemy)
+                |-- Resources (Contains all the routes and views to handle incoming requests)
+    |-- utils (Contains utility shared across modules)
+|-- tests (Unit/Integration tests for the application)
+|-- app.sh (Shell script used by the python OpenShift s2i image to run the application)
+|-- Dockerfile (Dockerfile for running the application locally using Docker)
+|-- requirements.txt (Libraries required by the project)
+```
+
+## Pre-requisites and Installation
+
+The application assumes you already have a working postgres DB with the required schema and tables and have the connection details in the .env file.
```

Comment:
Why is that assumption made? Is that because the microservice is (currently) meant to leverage the single MDS database that already exists?

If that's the case, I'm not sure about the phrasing here. It seems to indicate that you need postgres running on the host.

Reply:
Copy paste from existing readme, it's saying that the backend does nothing to stand up a database, that needs to be separate and you need to provide this service those credentials. Thats how i read it. Do you want some changes to it?

Reply:
My only concern is that it seems to apply when you're running the app on the host, but not when it's running in Docker. Certainly not a blocker for this PR, as it's copy/pasted. There's likely a need to have a full ticket to address the current state of all of our READMEs.

I'll mark this as resolved